scripts/jenkins/qa_openstack.sh

```diff
@@ -369,7 +369,7 @@ EOF
 openstack_client_ver=0
 
 if [ -x /usr/bin/openstack ]; then
-    openstack_client_ver=`openstack --version 2>&1 | cut -d" " -f2 | cut -d"." -f1`
+    openstack_client_ver=`openstack --version 2>&1 | grep -e ^WARNING -v | cut -d" " -f2 | cut -d"." -f1`
```

Comment:
Isn't dropping the `2>&1`  a better solution?
This way, the cut only gets the relevant input, and STDERR still shows the deprecation warning (and maybe others in the future).
The `grep -ev` just hides all future errors.